tests/test_numerical.py

```diff
@@ -156,6 +158,32 @@ def test_std_deviation_aliases(case):
     assert _.std_deviation is case
 
 
+@parametrize('minuend,subtrahend,expected', [
+    (10, 4, 6),
+    (-6, -4, -2),
+    (4, -10, 14),
+    (-10, 4, -14),
+    ('10', '5', 5),
+    (2, 0.5, 1.5),
+    (None, None, 0)
+])
+def test_subtract(minuend, subtrahend, expected):
+    assert _.subtract(minuend, subtrahend) == expected
+
+
+@parametrize('minuend,subtrahend,error_msg', [
+    ('abs', 4, 'unsupported operand type'),
+    (4, 'abc', 'unsupported operand type'),
+    ('abs', 'abc', 'unsupported operand type'),
+    ([1, 2, 3, 4], 4, 'unsupported operand type'),
+])
+def test_subtract_error(minuend, subtrahend, error_msg):
+    with pytest.raises(TypeError) as exc:
+        _.subtract(minuend, subtrahend)
+
+    assert error_msg in str(exc)
```

Comment:
We don't really need this assertion since we're not raising these exceptions ourselves.